Problems/2164/ans_2164_YH.py

```diff
@@ -0,0 +1,51 @@
+#https://www.acmicpc.net/problem/2164
+######
+class Node:
+    def __init__(self,data):
+        self.data = data
+        self.next = None
+        self.prev = None
+        
+class queue:
+    def __init__(self):
+        self.front = None
+        self.rear = None
+        self.size = 0
+    def qsize(self):
+        return self.size
+    def enqueue(self, data):
+        if not self.size:
```

Comment:
```suggestion
        if self.size==0:
```
굳이 integer를 boolean으로 처리해서 머리 아플 필요가 있을까?
특히 이 구문 같이 queue가 비어있으면~ 이라는 표현을 하고싶은 경우라면..